examples/platform/silabs/main.cpp

```diff
@@ -71,6 +71,18 @@ int main(void)
 
 void application_start(void * unused)
 {
+
```

Comment:
this implementation is changed in latest code , use latest code

Reply:
Done